src/pages/Register/Register.js

```diff
@@ -1,7 +1,189 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './Register.scss';
 const Register = () => {
-  return <div />;
+  // 초기값 세팅 = 아이디, 비밀번호, 비밀번호 확인, 이름, 생년원일, 휴대폰 번호
+  const [person, setPerson] = useState({
+    id: '',
+    pw: '',
+    pwConfirmed: '',
+    name: '',
+    birth: '',
+    phone: '',
+  });
+
```

Comment:
이후에 구조분해할당을 해서 사용하면 좀 더 편리하게 사용할 수 있습니다.
```suggestion
  const [person, setPerson] = useState({
    id: '',
    pw: '',
    pwConfirmed: '',
    name: '',
    birth: '',
    phone: '',
  });
  const { id, pw, pwConfirmed, name, birth, phone } = person;
```